desk/app/channels.hoon

```diff
@@ -587,7 +747,7 @@
       [?(%v0 %v1) ~]                        ?>(from-self cor)
       [?(%v0 %v1) %unreads ~]               ?>(from-self cor)
       [?(%v0 %v1) =kind:c ship=@ name=@ ~]  ?>(from-self cor)
-      [?(%v0 %v1) %said =kind:c host=@ name=@ %post time=@ reply=?(~ [@ ~])]
+      [?(%v0 %v1 %v2) %said =kind:c host=@ name=@ %post time=@ reply=?(~ [@ ~])]
```

Comment:
The `%v2` case will give you a different initial result (from `+ca-said`, which should become `+ca-said-1` and `+ca-said-2` separately) right?